src/components/Book.js

```diff
@@ -3,30 +3,49 @@ import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/books';
 
-const Book = ({ title, author, id }) => {
+const Book = ({
+  title, author, id, category,
+}) => {
   const dispatch = useDispatch();
```

Comment:
- [ ] Your app's current design does not match to the given wireframe. Make sure that all elements in design are added. Kindly take a look at the below screenshots to see differences and fix them 👇 
- Your design 👇 
![Screenshot (282)](https://user-images.githubusercontent.com/67087664/175526943-bcdfbda4-3bdd-4c0e-a485-6a19da3b1014.png)
- Original design 👇 
![originale](https://user-images.githubusercontent.com/67087664/175526994-49592a2e-bed7-4025-b61a-7f46e7394343.png)